lib/lightning/cw_lightning.dart

```diff
@@ -0,0 +1,69 @@
+part of 'lightning.dart';
+
+class CWLightning extends Lightning {
+
+  @override
+  String formatterLightningAmountToString({required int amount}) =>
+      bitcoinAmountToString(amount: amount * 100000000);
+
+  @override
+  double formatterLightningAmountToDouble({required int amount}) =>
+      bitcoinAmountToDouble(amount: amount * 100000000);
+
+  @override
+  int formatterStringDoubleToLightningAmount(String amount) =>
+      stringDoubleToBitcoinAmount(amount * 100000000);
+
+  WalletService createLightningWalletService(
+      Box<WalletInfo> walletInfoSource, Box<UnspentCoinsInfo> unspentCoinSource) {
+    return LightningWalletService(walletInfoSource, unspentCoinSource);
+  }
+
+  @override
+  List<LightningReceivePageOption> getLightningReceivePageOptions() =>
```

Comment:
please make sure that building Monero.com won't cause any issues with this

Reply:
did you check this?

Reply:
checking now

Reply:
it works but I now see your commit fixing the issues 😅